packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoWorkflowStepList/WfoWorkflowStepList.tsx

```diff
@@ -17,18 +19,60 @@ export interface WfoWorkflowStepListProps {
     isTask: boolean;
 }
 
+interface SubscriptionDeltaState {
+    subscription: object;
+}
+
 export const WfoProcessRawData = ({ processId }: { processId: string }) => {
     const { data, isFetching } = useRawProcessDetails(processId);
     return isFetching ? <WfoLoading /> : <WfoJsonCodeBlock data={data || {}} />;
 };
 
+export const WfoProcessSubscriptionDelta = ({
+    processId,
+}: {
+    processId: string;
+}) => {
+    const { data, isFetching } = useRawProcessDetails(processId);
+
+    const oldText =
+        data &&
+        'current_state' in data &&
+        'subscription' in (data.current_state as object)
+            ? (data.current_state as SubscriptionDeltaState).subscription
+            : '';
+
+    const newText =
+        data &&
+        'current_state' in data &&
+        '__old_subscriptions__' in (data.current_state as object)
```

Comment:
Is it possible to do something like

const currentState = data.current_state as {... __old_state__ .. } so you don't need to use the ts-ignore?

Reply:
Ts ignores are gone.